### ISSUE TITLE
[OFFICIAL] Bank-Vaults Vault Secrets Webhook

### ISSUE THREAD

**Issue by sagikazarmark** [artifacthub/hub#3211]
In Artifact Hub, the `official` status means that the publisher **owns the software** a package primarily focuses on. If we consider the *example* of a [chart used to install Consul](https://artifacthub.io/packages/helm/hashicorp/consul), to obtain the `official` status the publisher should be the owner of the Consul software (*HashiCorp* in this case), not just the chart. Similarly, a [Tekton task used to perform operations on Google Cloud](https://artifacthub.io/packages/tekton-task/tekton-catalog-tasks/gcloud) would need to be published by *Google* to be marked as `official`. In the case of a MySQL operator, only one published by *MySQL/Oracle* would be considered `official`.

The `official` status can be granted at the **repository or package level**. When it is granted for a repository, all packages available on it will display the `official` badge, so all packages in the repository **must** be official. If only some of the packages in your repository are official, please list them in the `Official packages` field below.

- **Repository name** *(in `artifacthub.io`)*: vault-secrets-webhook
- **Official packages** *(leave empty if all packages in the repository are official)*:
- **Project URL:** https://github.com/bank-vaults/vault-secrets-webhook
- **Is the publisher a CNCF project? (graduated, incubating or sandbox):** No
- **Source code URL** *(if available)*: https://github.com/bank-vaults/vault-secrets-webhook
- **Other relevant URLs:** *if there are any other urls that can help us with the verification process, please include them here*

**Before applying for this status, please make sure your repository complies with the following requirements:**

- [x] The repository has already obtained the [Verified Publisher](https://artifacthub.io/docs/topics/repositories/#verified-publisher) status.
- [x] The user requesting the status is the publisher of the repository in Artifact Hub, or belongs to the organization publishing it.
- [x] All official packages available in the repository provide a `README.md` file with some documentation that can be displayed on Artifact Hub.

**Comment by sagikazarmark** [comment]
Out of curiosity: if the package was named something different (eg. secrets-webhook) and supported other providers, not just Vault, would that be a potential candidate for official status? (This is what we plan to do with it in the long-run).

Although there are examples in the documentation what wouldn't be considered for official status, the highlighted section ("owns the software") puts an emphasis on owning the package itself, not what it integrates with which is true for both this component and the vault operator. It may be beneficial to slightly change that, because right now I think it's sending the wrong message. At least it looks like ownership is far from enough for official status.

**Comment by tegioz** [comment]
> Out of curiosity: if the package was named something different (eg. secrets-webhook) and supported other providers, not just Vault, would that be a potential candidate for official status? (This is what we plan to do with it in the long-run).

Yes, not being focused exclusively on Vault and with a different name would be fine. More so if the name made it clear that it is a Bank Vaults official artifact (someone else could publish a similar artifact based on your software, but it wouldn't be official).

The problem with generic names is that there could be (now or eventually) multiple packages named similarly or identically with a similar purpose. Marking all of them as official would be confusing for users and would probably defy the point of the official flag, and maybe no option would deserve that status over the others (being first shouldn't be enough probably). But having an official `bank-vaults-operator` (or any other name that uniquely identifies your software) would be awesome 🙂

> Although there are examples in the documentation what wouldn't be considered for official status, the highlighted section ("owns the software") puts an emphasis on owning the package itself, not what it integrates with which is true for both this component and the vault operator. It may be beneficial to slightly change that, because right now I think it's sending the wrong message. At least it looks like ownership is far from enough for official status.

Thanks, we'll try to improve that part!
